nginx-nodejs-redis/web/server.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const redis = require('redis');
+const app = express();
+const redisClient = redis.createClient({
+  host: 'redis',
+  port: 6379
+});
+
+app.get('/', function(req, res) {
+    redisClient.get('numVisits', function(err, numVisits) {
+        numVisitsToDisplay = parseInt(numVisits) + 1;
+        if (isNaN(numVisitsToDisplay)) {
+            numVisitsToDisplay = 1;
+        }
+        res.send('Number of visits is: ' + numVisitsToDisplay);
```

Comment:
Use the hostname defined in the compose file to display which node is responding

```suggestion
const os = require('os');
const app = express();
const redisClient = redis.createClient({
  host: 'redis',
  port: 6379
});

app.get('/', function(req, res) {
    redisClient.get('numVisits', function(err, numVisits) {
        numVisitsToDisplay = parseInt(numVisits) + 1;
        if (isNaN(numVisitsToDisplay)) {
            numVisitsToDisplay = 1;
        }
        res.send(os.hostname() +': Number of visits is: ' + numVisitsToDisplay);
```